### ISSUE TITLE
Android-reported "User data" almost twice as large as Signal's "Storage usage"

### ISSUE THREAD

**Issue by sbraz** [signalapp/Signal-Android#13758]
### Guidelines

- [X] I have searched searched open and closed issues for duplicates
- [X] I am submitting a bug report for existing functionality that does not work as intended
- [X] This isn't a feature request or a discussion topic

### Bug description

Hello,
Signal currently reports about 40 GiB under "Storage usage":
<img src="https://github.com/user-attachments/assets/aff927fa-7f8f-4f6f-abc7-eac231c3ae4d" width="400">

However, Android shows that it actually uses 71 GiB, that's almost twice as much:
<img src="https://github.com/user-attachments/assets/48efdf2b-a6b5-4d67-9662-fd29c52b65fc" width="400">

I understand that part of the issue could be that Signal's data was copied from another phone and I am hitting #11509 so I started with a huge amount of space used.
However, even after removing a lot of large duplicate files and seeing Signal's "Storage usage" greatly reduced, Android still shows that it is using way too much space.

This problem looks very much like #9634 which was resolved a long time ago, could they be similar?

### Screenshots
Embedded above.

### Device

Pixel 8a

### Android version

14

### Signal version

7.20.2

### Link to debug log

https://debuglogs.org/android/7.20.2/a16ce2e5f411bcba4ed3e73163933818985d0ee0f661c0b596d80609af0de543

**Comment by sbraz** [comment]
> As evidence, if you uninstall the app and reinstall it, the size will be less including attachments.

I might have missed something but I uninstalled the app, reinstalled it and I lost all data. Is Android usually supposed to keep data when apps are uninstalled?

**Comment by mosbat** [comment]
> > As evidence, if you uninstall the app and reinstall it, the size will be less including attachments.
> 
> I might have missed something but I uninstalled the app, reinstalled it and I lost all data. Is Android usually supposed to keep data when apps are uninstalled?

No, usually, everything is wiped when an app is removed unless the app enables you to download something into a different directory. 

I meant if you re-install the app assuming you got the same attachments downloaded again, the app size will be lesser than if you had the app on your phone for a very long period of time, accumulating updates and such.

**Comment by sbraz** [comment]
> I meant if you re-install the app assuming you got the same attachments downloaded again

I don't understand how one could do this however since uninstalling the app deleted all data?

> if you had the app on your phone for a very long period of time

I've had it for a few days, I transferred data from another phone. Do you mean that deleting files from the storage doesn't have the desired effect?
